---
frame_id: 'map'
take_off_altitude: 1.5
landing_position: [2.25, 1.75, 1.0, 0.0, 0.0, 0.0, 1.0]
loop_waypoints: True
waypoints:
    - [2.25,  -1.75, 2.0, 0.0, 0.0, 0.966, 0.259]
